test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1206,6 +1206,9 @@ def run_copy(self, hosts=None, src=None, dest=None, sudo=False, uid=None,
                            recursive=recursive, runas=runas)
             if ((uid is not None and uid != self.get_current_user()) or
                     gid is not None):
+                if dest == self.get_pbs_conf_file(targethost):
+                    uid = pwd.getpwnam('root')[2]
+                    gid = pwd.getpwnam('root')[3]
```

Comment:
Should we use 'root' or str(ROOT_USER)?

Reply:
We should use 'root' since it is used in the pw database of most *nix systems.

Reply:
Yeah, but we've abstracted it out in our PTL interface.  Why not use our interface?

Reply:
root is root and I didn't see the need to abstract it.

Reply:
I also tested the change and got this:
    uid = pwd.getpwnam(str(ROOT_USER))[2]
NameError: name 'ROOT_USER' is not defined

Reply:
Thanks for testing it.  pbs_testusers must not be imported.  Yeah, just use root.

Reply:
Yes I'll need to do add ROOT_USER in the line below:
from ptl.utils.pbs_testusers import PBS_ALL_USERS, PbsUser, ROOT_USER
Ok I'll keep 'root'.